dds2val/ddsprovider.py

```diff
@@ -0,0 +1,53 @@
+import logging
+import os
+import signal
+import asyncio
+from pathlib import Path
+from ddsproviderlib import helper
+
+log = logging.getLogger("ddsprovider")
+
+async def main():
+    """Perform the main function activities."""
+    logging.basicConfig(level=logging.DEBUG)
```

Comment:
We seem to ignore `LOG_LEVEL` that is set in Dockerfile. Should we possibly use INFO as default level.